### PR TITLE
fix (docs): renamed platform-related terms

### PR DESCRIPTION
**Pull request by mmaltsev-ct**
#### Summary

Renamed Platform-related terms following the rebranding.

#### Description

With a new Portfolio strategy for the company, some renaming of the used terms was done. For instance, the term "Platform" was replaced with the term "Composable Commerce".

As a result, all "Platform"-related text was replaced in the Custom Application documentation. The affected sentences were rephrased accordingly. Used the [existing PR](https://github.com/commercetools/commercetools-docs/pull/2007) for the rest of the documentation as a reference.

### CODE REVIEW

File: website/src/content/concepts/merchant-center-api.mdx
Comment (melindafekete) [inline website/src/content/concepts/merchant-center-api.mdx:29]:
```suggestion
The Merchant Center and the Merchant Center API Gateway are available in the same cloud Regions where Composable Commerce runs.
```

File: website/src/content/concepts/oauth-scopes-and-user-permissions.mdx
Comment (melindafekete) [inline website/src/content/concepts/oauth-scopes-and-user-permissions.mdx:7]:
```suggestion
Custom Applications can use any of the available Composable Commerce APIs. This allows you to extend the functionality and build custom features for any of the Composable Commerce APIs and the Merchant Center.
```

File: website/src/content/what-is-a-custom-application.mdx
Comment (melindafekete) [inline website/src/content/what-is-a-custom-application.mdx:17]:
```suggestion
* Extend the functionality of the Composable Commerce APIs and the Merchant Center
```

File: website/src/content/what-is-a-custom-application.mdx
Comment (melindafekete) [inline website/src/content/what-is-a-custom-application.mdx:19]:
```suggestion
* Integrate Composable Commerce and the Merchant Center with external services
```

File: website/src/content/what-is-a-custom-application.mdx
Comment (melindafekete) [inline website/src/content/what-is-a-custom-application.mdx:18]:
```suggestion
* Tailor to specific business requirements and use-cases
```

**Review by melindafekete — CHANGES_REQUESTED** [review]

**Comment by melindafekete** [comment]
hey @mmaltsev-ct thank you so much for this beauty 🤗  👏 

It's mostly good to go, just a couple of minor things. I suggested some content changes to the existing text too, I know it's not the main focus on the PR, but since we're already changing those lines, I think we might as well... Hopefully not too much trouble. Thank you!!

File: website/src/content/index.mdx
Comment (melindafekete) [reply website/src/content/index.mdx:61]: This is the only place where it's singular (at least amongst the changed files), so we could make this one plural to be consistent.

**As to when/why:**
We should definitely use the plural version in cases where the old 'platform' phrase would refer to the whole of commercetools including the Change History API, Machine Learning API, soon Checkout Anywhere, and so on.

I also prefer the plural in most cases when talking about the HTTP API alone. Even though it's technically a single API for now, the plural form helps reinforce the message that you're not required to use the whole thing. You could just be using the Carts API, the Products API, or the Discounts API. Probably less work for us later on when we split out Discounts.

That said, the plural/singular versions are currently quite inconsistent in the main docs repo. Platform API and Platform APIs were used across the docs to mean the same thing, and because of the volume of the changes we had to make in that repo, we decided to not focus too much on that now. 

So in short, **let's go with plural**, but if it's not perfect for everywhere that's also ok. What do you think @emmenko?

@adinakleine do you have anything to add?

Comment (adinakleine) [reply website/src/content/index.mdx:61]: Thanks for your suggestion @melindafekete. I do agree. 💯

Comment (emmenko) [reply website/src/content/index.mdx:61]: Should we add this rule to our writing style rules?

Comment (adinakleine) [reply website/src/content/index.mdx:61]: > Should we add this rule to our writing style rules?

@melindafekete Would be great if you could care for this. 🙇‍♀️

File: website/src/content/what-is-a-custom-application.mdx
Comment (mmaltsev-ct) [reply website/src/content/what-is-a-custom-application.mdx:19]: @melindafekete thanks for the suggestion! What bothers me is that if we do it like that, it would be the only place where we separate Composable Commerce and the MC, since in other places we say "Composable Commerce APIs and the Merchant Center". What do you think?

File: website/src/content/index.mdx
Comment (mmaltsev-ct) [reply website/src/content/index.mdx]: Changed to plural. Marking this thread as resolved.

File: website/src/content/what-is-a-custom-application.mdx
Comment (melindafekete) [reply website/src/content/what-is-a-custom-application.mdx:19]: Good point, let's keep the original change 👍

File: website/src/content/api-reference/application-config.mdx
Comment (melindafekete) [reply website/src/content/api-reference/application-config.mdx:573]: Yes, that's perfect, just missing a `the`, so: ```..that can be used in the same Regions (Europe and North America) as the Composable Commerce APIs?```

Comment (mmaltsev-ct) [reply website/src/content/api-reference/application-config.mdx]: Thank you, fixed it.

**Review by melindafekete — APPROVED** [review]

**Comment by mmaltsev-ct** [comment]
Hey @tdeekens, I think we're good to go with this PR, but I need a review from someone with write access (I don't have one).

**Review by adinakleine — APPROVED** [review]
Lgtm 👍  Thanks!

**Review by tdeekens — APPROVED** [review]

**Pull request merged**
